packages/cli/src/commands/ps/autoscale/enable.ts

```diff
@@ -26,12 +27,25 @@ export default class Enable extends Command {
     ux.action.start('Enabling dyno autoscaling')
 
     const [appResponse, formationResponse] = await Promise.all([
-      this.heroku.get<Heroku.App>(`/apps/${flags.app}`),
-      this.heroku.get<Heroku.Formation[]>(`/apps/${flags.app}/formation`),
+      this.heroku.get<App>(`/apps/${flags.app}`, {
+        headers: {
+          Accept: 'application/vnd.heroku+json; version=3.sdk',
+        },
+      }),
+      this.heroku.get<Formation[]>(`/apps/${flags.app}/formation`, {
+        headers: {
+          Accept: 'application/vnd.heroku+json; version=3.sdk',
+        },
+      }),
     ])
     const app = appResponse.body
     const formations = formationResponse.body
     const webFormation = formations.find((f: any) => f.type === 'web')
+
+    if (app.generation === 'fir') {
+      throw new Error('Autoscaling is unavailable for apps in this space. See https://devcenter.heroku.com/articles/generations.')
```

Comment:
Has cx signed off on this error message? @SandyPantsLai.. I think it differs from the approved message in https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE0000201VQxYAM/view

Reply:
I'm seeing the same text I have here in the work item... Am I missing something?